src/Cubes/TransformedCubes.jl

```diff
@@ -5,12 +5,15 @@ using DiskArrayTools: diskstack, DiskArrayTools
 
 function Base.map(op, incubes::YAXArray...)
     axlist = copy(caxes(incubes[1]))
+    chunks = incubes[1].chunks
```

Comment:
Is there a difference between using c.chunks and eachchunk(c) ? 
It seems that you are using both to access the chunks of a cube.

Reply:
If c is a YAXArray then there is no difference but in general we should prefer the accessor function instead of accessing the field directly